.github/workflows/ci.yml

```
@@ -42,4 +35,58 @@ jobs:
+          /Applications/vibe.app/Contents/MacOS/vibe &> app_output.log &
+          APP_PID=$!
+          sleep 10
+          kill $APP_PID
+          if grep -i "error" app_output.log; then
+            echo "Error found in application output."
+            exit 1
+          fi
+          set -e
+      - name: Upload App Logs
+        if: always()
+        uses: actions/upload-artifact@v3
```

actionlint (1.7.7)

the runner of "actions/upload-artifact@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)